README.md

```diff
@@ -2,391 +1,310 @@
+See full documentation for Learnosity API init options, methods and events in the [Learnosity reference site](https://reference.learnosity.com/).
 
-## Building with Maven
+### **Technical use-cases documentation**
+Find guidance on how to select a development pattern and arrange the architecture of your application with Learnosity, in the [Technical Use-Cases Overview](https://help.learnosity.com/hc/en-us/articles/360000757777-Technical-Use-Cases-Overview).
 
-Run ```mvn package``` in your root directory (where pom.xml is located). This will put the light and all jar in the correct folders.
+### **Deciding what to build or integrate**
+Get help deciding what application functionality to build yourself, or integrate off-the-shelf with the [Learnosity "Golden Path" documentation](https://help.learnosity.com/hc/en-us/articles/360000754578-Recommended-Deployment-Patterns-Golden-Path-).
 
-## Using with a maven file based repository
+### **Key Learnosity concepts**
+Want more general information about how apps on Learnosity actually work? Take a look at our [Key Learnosity Concepts page](https://help.learnosity.com/hc/en-us/articles/360000754638-Key-Learnosity-Concepts).
 
-1.) Download the 'light' jar and the learnosity project pom file (pom.xml) into a temporary directory (say /home/temp)
+### **Glossary**
+Need an explanation for the unique Learnosity meanings for Item, Activity and Item bank? See our [Glossary of Learnosity-specific terms](https://help.learnosity.com/hc/en-us/articles/360000754838-Glossary-of-Learnosity-and-Industry-Terms).
 
-2.) Add a directory to your maven project to build the file-based repository into (e.g. ./learnosity-sdk-repo)
+[(Back to top)](#table-of-contents)
 
-3.) Install the learnosity SDK with maven into that directory by running the following (modify locations as appropriate):
-```bash
-mvn deploy:deploy-file -DgroupId=learnositysdk -DartifactId=learnositysdk -Dversion=0.16.3 -Durl=file:./learnosity-sdk-repo/ -DrepositoryId=learnosity-sdk-repo -DupdateReleaseInfo=true -Dfile=/home/temp/learnositysdk-0.16.3.jar -DpomFile=/home/temp/pom.xml
-```
+## Contributing to this project
 
-4.) Add the directory as a file based repository in your project pom.xml:
-```xml
-<repositories>
-    <repository>
-        <id>learnosity-sdk-repo</id>
-        <url>file://${project.basedir}/learnosity-sdk-repo</url>
-    </repository>
-</repositories>
-```
+### Adding new features or fixing bugs
+Contributions are welcome. See the [contributing instructions](CONTRIBUTING.md) page for more information. You can also get in touch via our support team.
+
+[(Back to top)](#table-of-contents)
 
-5.) Don't forget to commit your file-based repository
+## License
+The Learnosity Java SDK is licensed under an Apache 2.0 license. [Read more](LICENSE.md).
 
-## Tracking
-In version v0.16.0 we introduced code to track the following information by adding it to the request being signed:
+[(Back to top)](#table-of-contents)
+
+## Usage tracking
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
## Usage tracking

```